SPECS/cyrus-sasl-ldap/cyrus-sasl-ldap.spec

```diff
@@ -0,0 +1,214 @@
+%define _soversion 3
+%global _plugindir2 %{_libdir}/sasl2
+%global _base_name cyrus-sasl
+
+Summary:        Cyrus Simple Authentication Service Layer (SASL) library
+Name:           %{_base_name}-ldap
```

Comment:
The new spec will require updates to `cgmanifest.json` and the licenses file, but the PR checks should details and (in some cases) full command lines to run to resolve the check.